tests/test_nubank_client.py

```diff
@@ -683,19 +683,56 @@ def test_get_qr_code():
     assert uid != ''
     assert isinstance(qr, QRCode)
 
-
-@pytest.mark.parametrize("http_status", [
+not_accepted_http_codes = [
     100, 101, 102, 103,
     201, 202, 203, 204, 205, 206, 207, 208, 226,
     300, 301, 302, 303, 304, 305, 306, 307, 308,
     400, 401, 402, 403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 420, 421, 422,
     423,
     424, 426, 428, 429, 431, 440, 444, 449, 450, 451, 495, 496, 497, 498, 499,
     500, 501, 502, 503, 504, 505, 506, 507, 508, 509, 510, 511, 520, 521, 522, 523, 524, 525, 526, 527, 530, 598
-])
+]
+
+@pytest.mark.parametrize("http_status", not_accepted_http_codes)
```

Comment:
Lets not mix double quotes and single quotes, lets use just single ones 😄

Reply:
Single and double quotes are mixed all over the repo. If you want I can standardize and put it along in this PR.

Reply:
I agree, maybe fix all the repo should be a separate PR also configuring a lint step on CI